src/app/shared/font-picker/font-picker.component.html

```diff
@@ -0,0 +1,9 @@
+<button mat-icon-button [matMenuTriggerFor]="fontMenu" matTooltip="Font Styles">
+ <h2>Aa</h2>
+</button>
+
+<mat-menu  #fontMenu="matMenu" x-position="before">
+      <mat-option (click)="onFontChange('my-first-font')" value="my-first-font">Monospace</mat-option>
+      <mat-option (click)="onFontChange('my-second-font')" value="my-second-font">Anton</mat-option>
+      <mat-option (click)="onFontChange('my-third-font')" value="my-third-font">Roboto</mat-option>
+</mat-menu>
```

Comment:
Use for loop to add options for fonts using an object. They should also be properly renamed instead of `my-font`. Ensure they are in alphabetical order.